.github/actions/test-dotnet/replace-xunit-maxParallelThreads.ps1

```diff
@@ -0,0 +1,12 @@
+param ($MaxParallelThreads)
+
+Write-Output "Replacing maxParallelThreads in xunit.runner.json files to $MaxParallelThreads."
+
+$counter = 0
+
+Get-ChildItem "*/**/xunit.runner.json" -Recurse | ForEach {
+     $counter++
+     (Get-Content $_) | ForEach  {$_ -Replace '"maxParallelThreads":\s*([-\d]*)', "`"maxParallelThreads`": $MaxParallelThreads"} | Set-Content $_
+}
```

Comment:
It would be more natural for pwsh to use `| Measure-Object` instead and the use the result's `Count` property.

Reply:
`ForEach` already can return a `Count` but that didn't work properly at all. Does I needed to get into the dark art of variable assignment, just for you: https://superuser.com/questions/1456576/powershell-incorrect-array-count-when-piping-foreach-object